.github/workflows/testing.yml

```diff
@@ -0,0 +1,171 @@
+      github.event_name == 'workflow_dispatch'
+
+    runs-on: ubuntu-latest
+
+    env:
+      COVERAGE_FILE: 'coverage.out'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        golang:
+          - '1.20'
+          - 'stable'
+        sdk-path:
+          - 'release/linux/x86_64/2.10/'
+        sdk-version:
+          - 'sdk-gc64-2.10.8-0-r598.linux.x86_64'
+        coveralls: [ false ]
+        include:
+          - golang: '1.20'
+            sdk-path: 'release/linux/x86_64/2.11/'
+            sdk-version: 'sdk-gc64-2.11.1-0-r598.linux.x86_64'
+            coveralls: false
+          - golang: 'stable'
+            sdk-path: 'release/linux/x86_64/2.11/'
+            sdk-version: 'sdk-gc64-2.11.1-0-r598.linux.x86_64'
+            coveralls: false
+          - golang: '1.20'
+            sdk-path: 'release/linux/x86_64/3.0/'
+            sdk-version: 'sdk-gc64-3.0.0-0-gf58f7d82a-r23.linux.x86_64'
+            coveralls: false
+          - golang: 'stable'
+            sdk-path: 'release/linux/x86_64/3.0/'
+            sdk-version: 'sdk-gc64-3.0.0-0-gf58f7d82a-r23.linux.x86_64'
+            coveralls: true
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Tarantool ${{ matrix.sdk-version }}
+        run: |
+          ARCHIVE_NAME=tarantool-enterprise-${{ matrix.sdk-version }}.tar.gz
+          curl -O -L https://${{ secrets.SDK_DOWNLOAD_TOKEN }}@download.tarantool.io/enterprise/${{ matrix.sdk-path }}${ARCHIVE_NAME}
+          tar -xzf ${ARCHIVE_NAME}
+          rm -f ${ARCHIVE_NAME}
+
+      - name: Setup golang for the connector and tests
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{matrix.golang}}
+
+      - name: Run regression tests
+        run: |
+          source tarantool-enterprise/env.sh
+          go test -v -count=1 -shuffle=on ./...
+          go test -race -v -count=1 -shuffle=on ./...
+
+      - name: Run tests, collect code coverage data and send to Coveralls
+        if: ${{ matrix.coveralls }}
+        env:
+          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          source tarantool-enterprise/env.sh
+          go clean -testcache
+          go get golang.org/x/tools/cmd/cover
+          go test ./... -v -count=1 -shuffle=on -covermode=atomic -coverprofile=${COVERAGE_FILE}
+          go tool cover -func=${COVERAGE_FILE}
+          go get github.com/mattn/goveralls
+          go install github.com/mattn/goveralls
+          goveralls -coverprofile=${COVERAGE_FILE} -service=github
+
+  testing_mac_os:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.repository) ||
+      (github.event_name == 'workflow_dispatch')
```

Comment:
Please, copy conditions from `run-tests-ee`. It allows to run CI for pull requests from an external repositories correctly.

Reply:
Updated.